proto/authzed/api/validation_test/tuples_test.go

```diff
@@ -369,3 +423,22 @@ func TestWildcardSubjectRelationEmpty(t *testing.T) {
 	}
 	require.NoError(t, subRef.HandwrittenValidate())
 }
+
+func generateMap(length int) map[string]any {
+	output := make(map[string]any, length)
+	for i := 0; i < length; i++ {
+		random := randString(32)
+		output[random] = random
+	}
+	return output
+}
+
+var randInput = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+func randString(length int) string {
+	b := make([]rune, length)
+	for i := range b {
+		b[i] = randInput[rand.Intn(len(randInput))] //nolint:gosec
```

Comment:
Using randoms in tests can cause issues; I'd rather pre-generate a bunch of values

Reply:
After discussion, this is fine, since it approximates context usage